src/vectors.S

```diff
@@ -0,0 +1,112 @@
+.section .text.init
+.global _run_guest_vector,_guest_exit_vector
+
+// t5 -> cached sstatus
+// restore vectors for guest
```

Comment:
Nit (here and elsewhere): Capitalize first letter of sentence if applicable.

Reply:
fixed